### PR TITLE
Fix: SmartServoClass::getPosition return value

### PR DESCRIPTION
**Pull request by metanav**
This PR fixes the issue #97.

### CODE REVIEW

File: src/lib/motors/SmartServo.cpp [hunk 1]
```diff
@@ -235,7 +235,18 @@ float SmartServoClass::getPosition(uint8_t const id)
     return -1.0f;
 
   mbed::ScopedLock<rtos::Mutex> lock(_mtx);
-  return positionToAngle(readWordCmd(id, REG(SmartServoRegister::POSITION_H)));
+  int position = readWordCmd(id, REG(SmartServoRegister::POSITION_H));
```
Comment (aentinger) [inline src/lib/motors/SmartServo.cpp:238]:
```suggestion
  int const position = readWordCmd(id, REG(SmartServoRegister::POSITION_H));
```

**Review by aentinger — CHANGES_REQUESTED** [review]
LGTM :+1: , only a minor improvement idea.

File: src/lib/motors/SmartServo.cpp (hunk 1 above)
Comment (metanav) [reply src/lib/motors/SmartServo.cpp]: Thanks, @aentinger! I have committed the suggested change.

**Review by metanav — COMMENTED** [review]
I think we are good now.

**Pull request merged**